src/features/landing/components/AuthSection/ConnectedForm/ConnectedForm.tsx

```diff
@@ -0,0 +1,51 @@
+import { TypedDocumentNode, gql, useMutation } from '@apollo/client';
+import React from 'react';
+
+import { RegisterItem } from '../../../types/authentication/form';
+import { AuthForm } from '../AuthForm';
+
+type SignupMutationVariables = {
+  FullName: string;
+  Email: string;
+  Password: string;
+};
+
+function ConnectedForm(): React.ReactElement {
+  const SIGNUP_MUTATION: TypedDocumentNode<SignupMutationVariables> = gql`
```

Comment:
this is incorrect place for mutation, this component shouldn't know about it, need move it to specific file where all mutations will be included